docs/src/app/components/pages/components/TimePicker/Page.js

```diff
@@ -33,6 +36,13 @@ const TimePickersPage = () => (
       <TimePickerExampleSimple />
     </CodeExample>
     <CodeExample
+      title="Inline TimePicker examples"
+      description={descriptions.inline}
```

Comment:
I'm wondering. Is react-docgen able to grab the description above the functionnal component? Try to remove this line.